web/models/group.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Pairmotron.Group do
+  use Pairmotron.Web, :model
+
+  schema "groups" do
+    field :name, :string
+    belongs_to :owner, Pairmotron.User
+    many_to_many :users, Pairmotron.User, join_through: Pairmotron.UserGroup
```

Comment:
Fun fact: since this association is going to be used in a `Ecto.Changeset.put_assoc` call and the `users_groups` table has timestamps, we need to explicitly join through the Ecto schema. If we were just going to manually insert UserGroup records manually (which is the plan), we could just join through the column name only. This is what's done in the many_to_many association on the User model, since we don't need to use put_assoc or cast_assoc from that end.